ci/cpu/upload_anaconda.sh

```diff
@@ -31,6 +33,11 @@ if [ "$UPLOAD_LIBCUDF" == "1" ]; then
   echo "Upload libcudf"
   echo ${LIBCUDF_FILE}
   anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} ${LABEL_OPTION} --skip-existing ${LIBCUDF_FILE}
+
+  test -e ${LIBCUDF_KAFKA_FILE}
+  echo "Upload libcudf_kafka"
+  echo ${LIBCUDF_KAFKA_FILE}
+  anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} ${LABEL_OPTION} --skip-existing ${LIBCUDF_KAFKA_FILE}
```

Comment:
We don't want to upload this for separate cuda versions like we do for libcudf, likely want to do it only once.